lib/instrumentation/modules/aws-sdk/sns.js

```diff
@@ -0,0 +1,102 @@
+const constants = require('../../../constants')
+
+const TYPE = 'messaging'
+const SUBTYPE = 'sns'
+const ACTION = 'PUBLISH'
```

Comment:
Should that be 'publish' or 'send' per https://github.com/elastic/apm/blob/master/specs/agents/tracing-instrumentation-messaging.md#typing ?

Reply:
Yup, `send` it is.  Change this variable, and changed the name string to use a hard coded `ACTION`.